engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/AvroUtils.scala

```diff
@@ -74,7 +51,9 @@ object AvroUtils extends Serializable {
   def valueSubject(topic: String): String =
     topic + "-value"
 
-  def createSchema(avroSchema: String): Schema =
+  def parseSchema(avroSchema: String): Schema =
     parser.parse(avroSchema)
 
+  def extractSchema(schemaMetadata: SchemaMetadata): Schema =
```

Comment:
Can you inline usage of this method? We need to see all usage of `parseSchema` to be sure that we don't parse too much...

Reply:
Done